packages/web-ui/app/page.tsx

```diff
@@ -1,39 +1,11 @@
-import Link from 'next/link';
-
-import { siteConfig } from '@/config/site';
-import { buttonVariants } from '@/components/ui/button';
+import { Chat } from '@/components/chat';
+import { Rooms } from '@/components/rooms';
 
 export default function IndexPage() {
   return (
-    <section className="container grid items-center gap-6 pb-8 pt-6 md:py-10">
-      <div className="flex max-w-[980px] flex-col items-start gap-2">
-        <h1 className="text-3xl font-extrabold leading-tight tracking-tighter md:text-4xl">
-          Beautifully designed components <br className="hidden sm:inline" />
-          built with Radix UI and Tailwind CSS.
-        </h1>
-        <p className="max-w-[700px] text-lg text-muted-foreground">
-          Accessible and customizable components that you can copy and paste
-          into your apps. Free. Open Source. And Next.js 13 Ready.
-        </p>
-      </div>
-      <div className="flex gap-4">
-        <Link
-          href={siteConfig.links.docs}
-          target="_blank"
-          rel="noreferrer"
-          className={buttonVariants()}
-        >
-          Documentation
-        </Link>
-        <Link
-          target="_blank"
-          rel="noreferrer"
-          href={siteConfig.links.github}
-          className={buttonVariants({ variant: 'outline' })}
-        >
-          GitHub
-        </Link>
-      </div>
-    </section>
+    <div className="flex h-full">
+      <Rooms />
+      <Chat />
```

Comment:
this is probably going to be more than the `Chat`. It will take different aspects according to our needs, it could be a `JoinChat`, a `Chat`, a `NoPermissions`

Maybe we can see if we should declare a more abstract component (probably layout) that can coordinate this side of the screen

Reply:
It was refactored.